modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -66,6 +66,7 @@
 			flex-grow: 1;
 			padding: var( --n-tabs-content-padding );
 			border-radius: var( --n-tabs-content-border-radius );
+			min-width: 0;
```

Comment:
This is the fix.
The elements were disappearing inside a flex element without min-width: 0.